lib/Rex/Pkg/FreeBSD.pm

```diff
@@ -69,9 +75,21 @@ sub is_installed {
   Rex::Logger::debug(
     "Checking if $pkg" . ( $version ? "-$version" : "" ) . " is installed" );
 
+  # only need -g if pkg is a glob
+  my $glob_arg = '';
+  if ( $pkg =~ /\*/ ) {
+
+    # back quote asterisks so the command can actually be used
+    $pkg =~ s/\*/\\\*/g;
+    $glob_arg = ' -g ';
```

Comment:
Are both leading and trailing spaces are needed for this to work?

Nit: I believe the trailing one would cause a double space below when continued with ` -e $pkg`, so it might be removed.

Reply:
Not really, but out of habit I always do this when putting together args stuff like to be overly careful.